.env.sample

```diff
@@ -29,4 +29,5 @@ S3_UPLOAD_REGION=""
 
 # Misc
 PROCESSING_FEE="26"
-CONFIRMATION_EMAIL_FROM=""
\ No newline at end of file
+CONFIRMATION_EMAIL_FROM=""
+INVOICE_LINK_DURATION_DAYS="7"
```

Comment:
env var for invoice link ttl. Made the corresponding changes when generating invoice and when fetching from db

Reply:
```suggestion
INVOICE_LINK_TTL_DAYS="7"
```
Naming suggestion